test/manual/api/structure_signature_api_tests.sh

```
@@ -0,0 +1,462 @@
+    echo "ERROR: No valid system found in API response. Available systems:" >&2
+    echo "$response" | jq '.' >&2
+    exit 1
+  else
+    echo "ERROR: Failed to fetch systems (status $status)" >&2
+    exit 1
+  fi
+}
+
+# ─── STRUCTURE TESTS ─────────────────────────────────────────────
+create_structure() {
+  local sys_info=$(get_first_system)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)